Snakefile

```diff
@@ -52,24 +57,23 @@ rule prepare_transport_data:
         temp_air_total="resources/temp_air_total_elec_s{simpl}_37.nc",
 
     output: 
-        nodal_energy_totals='resources/nodal_energy_totals.csv',
-        transport='resources/transport.csv',
-        avail_profile='resources/avail_profile.csv',
-        dsm_profile='resources/dsm_profile.csv',
-        nodal_transport_data='resources/nodal_transport_data.csv',
+        # nodal_energy_totals='resources/nodal_energy_totals.csv',
+        # transport='resources/transport.csv',
+        # avail_profile='resources/avail_profile.csv',
+        # dsm_profile='resources/dsm_profile.csv',
+        # nodal_transport_data='resources/nodal_transport_data.csv',
+        dummy_wildcard="resources/dummy{simpl}_{clusters}.nc"

```

Comment:
This output has the only purpose of using wildcards in the outputs as a link to the input.
The recommendation here is to avoid using input files with wildcards.

For debugging, we may hard specify the simpl and clusters wildcards in the input of this rule and remove the output dummy wildcard, but for long-term we need a slight restructuring and a procedure to avoid using wildcards in the input if the outputs do not need them

Reply:
I think I got your idea behind it to 80 %... Nonetheless, we need the outputs: nodal_energy_totals, transport, avail_profile, dsm_profile, nodal_transport_data as an input for prepare_sector_network.py, but now they are commented out. Maybe we should have a short discussion on this on Thursday

Reply:
I revised the script to leave the outputs, but I needed to hardcode the wildcards in the inputs.
On Thursday let's discuss.

By the way there are two ways to proceeed:
1. put the wildcards on all output values, e.g. 'resources/dsm_profile_{simpl}_{clusters}.csv'
2. remove the wildcards from the inputs, e.g. using add_electricity output (networks/elec.nc) for networks